src/common/components/profile-cover/_index.scss

```diff
@@ -26,7 +26,7 @@
     position: absolute;
     right: 10px;
     top: 10px;
-    z-index: 2;
+    z-index: 1;
```

Comment:
Check my comment on trello, please fix it in navbar code, because this (profile-cover) was not the cause of issue.

Reply:
I understood the problem. While I was making the changes on the navbar I added `z-index: 2` to the `.navbar` class name. And because the bottom side follow, favourite buttons wrapper also smae z-index value and it comes after navbar  in the stacking order buttons comes over the navbar elements. That was the problem. Increased the z-index of the navbar to  `3` and fixed @feruzm

Reply:
@kodcuserkan I merged this one, looks better fix, no? https://github.com/ecency/ecency-vision/pull/500

Reply:
@feruzm it is exactly what I did 1 week ago. But you commented as above `Check my comment on trello, please fix it in navbar code, because this (profile-cover) was not the cause of issue.` and then I rollback my commit as original and focused to navbar component and  did  my last changes. Both of them fine but I recommend merge both.